### ISSUE TITLE
Strange instrument bug.

### ISSUE THREAD

**Issue by steviator** [starg2/timidity41#5]
When using this soundfont in twsyng or timw32g, instrument 64 and 66 do not play, but if running timidity from the command line, those instruments do play. I wasted hours chasing my tail trying to figure out why these instruments don't want to play before finally coming to the conclusion that it's a bug that only shows up in the GUI parts of the application.

This represents a major show stopping regression from mainline Timidity, and obviously renders it completely useless, since it can't be relied upon to play a given instrument. 

[grandpa.zip](https://github.com/user-attachments/files/16391873/grandpa.zip)

**Comment by starg2** [comment]
Does the console window show any errors/warnings?

Please check if both CLI and GUI are configured in the same way. Note that the CLI version always reads `timidity.cfg` and doesn't respect the configuration set by the GUI.

**Comment by steviator** [comment]
Hi, thanks for the quick response. The config file is being explicitly specified with a full path name for both twsyng (in the GUI prefs) and on the command line for timidity. 

The relevant lines from my quite lengthy config file are as follows:

  064 %font "grandpa.sf2"  0 064 amp=7 # tune=.2 # Soprano Sax
  065 %font "grandpa.sf2"  0 065 amp=17 tune=.2 # Alto Sax
  066 %font "grandpa.sf2"  0 066 amp=9  tune=.2 # Tenor Sax
  067 %font "grandpa.sf2"  0 067 amp=15 tune=.2 # Baritone Sax

This works fine on timidity on Linux and with the CLI version of windows, as well as with a non-starg version of twsyng that I have reverted to in the meantime. 

It seems I may have slightly misreported the symptoms here, perhaps I was tired at the time - two of the four presets work and  two are silent in twsyng, and I can't say which ones for sure since I'm not in front of that machine right now, but if my memory serves me correctly, the broken presets were 064 (Soprano Sax) and 066 (Tenor Sax). I can say for sure that 065 (Alto Sax) was definitely working in all versions of timidity that I tried.

Once I'm safely ensconced in my hovel again, I'll get back to you with some example mid files and a confirmation of exactly which two instruments are the problematic ones, but I'd be very surprised to hear that all the instruments in that soundfont work for you in twsyng in testing.
